rqt_py_common/package.xml

```diff
@@ -37,6 +37,8 @@
   <run_depend>actionlib</run_depend>
   <run_depend>rosbag</run_depend>
 
+  <test_depend>pr2_moveit_config</test_depend>
```

Comment:
Instead of adding this test dependency I would suggest to modify the test to check for a more "common" package. Otherwise this package can't be released into a new distro until `pr2_moveit_config` has been released which is a huge restriction.